components/Cards/src/MDCCardExperimentalColorThemer.m

```diff
@@ -0,0 +1,35 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCCardExperimentalColorThemer.h"
+#import "MDCCard.h"
+#import "MDCCardCollectionCell.h"
+
+@implementation MDCCardExperimentalColorThemer
+
++ (void)applyExperimentalTheme:(MDCExperimentalColorScheme *)colorScheme toCard:(MDCCard *)card {
+  [card setBorderColor:colorScheme.borderColor forState:UIControlStateNormal];
+  [card setShadowColor:colorScheme.shadowColor forState:UIControlStateNormal];
+  card.backgroundColor = colorScheme.backgroundColor;
+}
+
++ (void)applyExperimentalTheme:(MDCExperimentalColorScheme *)colorScheme toCardCollectionCell:(MDCCardCollectionCell *)cell {
+  [cell setBorderColor:colorScheme.borderColor forState:MDCCardCellStateNormal];
+  [cell setShadowColor:colorScheme.shadowColor forState:MDCCardCellStateNormal];
```

Comment:
same here

Reply:
also do we care about the selected state too? how about the selected image color?

Reply:
Are they supposed to change when it's selected?

Reply:
the image tint color for the selected image should be part of the theming. As to what color, I presume it should be the selectionColor you set up. You should probably rebase if you don't have the latest merge from last week